tests/network/vmi_networking.go

```diff
@@ -415,7 +415,10 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 			deadbeafVMI, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), deadbeafVMI)
 			Expect(err).ToNot(HaveOccurred())
 
-			libwait.WaitUntilVMIReady(deadbeafVMI, console.LoginToAlpine)
+			const unregisterSlipImageWarning = "no Slirp network binding plugin image is set in Kubevirt config, using " +
+				"'quay.io/kubevirt/network-slirp-binding:20230830_638c60fc8' sidecar image for Slirp network binding configuration"
+			warnings := append(testsuite.TestRunConfiguration.WarningToIgnoreList, unregisterSlipImageWarning)
+			libwait.WaitUntilVMIReady(deadbeafVMI, console.LoginToAlpine, libwait.WithWarningsIgnoreList(warnings))
```

Comment:
I think the other way around should be done (I"m pretty sure I already wrote this comment, but I cannot find it). We should verify the warning is printed instead of ignoring it. That's the new logic of this PR.

Reply:
?

Reply:
We could be fine with making sure it is tested in unit tests.
Checking events in e2e here is borderline to me, it somehow relates a bit to checking logs.

But if it is easy to do, I guess no harm in having it.